database.js

```diff
@@ -13,7 +13,13 @@ function connect(){
 function getdb(){
```

Comment:
do we still need this if everything else from now on uses mongoose? If not delete it. Make sure no lines of code still uses this method